financial-services-accelerator/components/org.wso2.financial.services.accelerator.consent.mgt.extensions/src/main/java/org/wso2/financial/services/accelerator/consent/mgt/extensions/authorize/model/ExternalAPIPreConsentAuthorizeRequestDTO.java

```diff
@@ -29,15 +29,17 @@ public class ExternalAPIPreConsentAuthorizeRequestDTO {
     private String userId;
     private JSONObject requestParameters;
     private ExternalAPIConsentResourceRequestDTO consentResource;
+    private String consentFileContent;
 
     public ExternalAPIPreConsentAuthorizeRequestDTO(ConsentData consentData,
                                                     ExternalAPIConsentResourceRequestDTO consentResource,
-                                                    JSONObject requestParameters) {
+                                                    JSONObject requestParameters, String consentFileContent) {
 
         this.consentId = consentData.getConsentId();
         this.userId = consentData.getUserId();
         this.consentResource = consentResource;
         this.requestParameters = requestParameters;
+        this.consentFileContent = consentFileContent;
```

Comment:
Couldn't we make consent file content also as a parameter inside consent resource?

Reply:
Fixed in https://github.com/wso2/financial-services-accelerator/pull/566/commits/a7b2880f7bb32f795ffc3ff4195cdb4a6b1ee951